test/inductor/test_flex_attention.py

```diff
@@ -3728,6 +3728,8 @@ def forward(self, L_query_: "f64[2, 2, 128, 4]", L_key_: "f64[2, 2, 128, 4]", L_
         l_block_mask_full_q_num_blocks = L_block_mask_full_q_num_blocks
         l_block_mask_full_q_indices = L_block_mask_full_q_indices
 
+        get_device_capability = torch.cuda.get_device_capability('cuda');  get_device_capability = None
```

Comment:
I am surprised this shows up in the graph module

Reply:
Should probably be marked as a constant, yea. although it will get traced out in tracing so only affects pre grad.

Reply:
Weird, it shows up in the graph module when I use:

```python
is_sm_100 = torch.cuda.get_device_capability("cuda") == (10, 0)
```

However, it doesn't show up when I do this:

```python
should_enforce_mem_layout = (
        gemm_precision in fp8_dtypes
        and torch.version.cuda is not None
        and torch.cuda.get_device_capability("cuda") >= (8, 9)
        and torch.cuda.get_device_capability("cuda") < (10, 0)
    )
```

Using the latter approach now.